### PR TITLE
Cu 864dpwv8d criar e setupar inicialmente o bosch planner app igor oliveira

### PR DESCRIPTION
**Pull request by iguoliveira**
##### Description of changes

- [x] Why (Why did you make this pull request): Initial setup of planner app (server and web)
- [x] Brief description of the changes: Modified web and server templates to fulfill initial planner's necessities
- [x] Possible breakouts (Possible places where things could break): Error with .env file, not locating linking path. A possible local error.

### CODE REVIEW

**Review by vitormiura — CHANGES_REQUESTED** [review]
**Forgot to change web and server ports, for server, need to change the PORT in index.ts to process.env.VITE_WEB_PLANNER_PORT | | 3002.**
image
**Is good too to setup cors too with planner web env, just add "VITE_WEB_PLANNER_PORT" instead of "VITE_WEB_TEMPLATE_PORT"**
image
**In web is necessary to change the port too, remove "VITE_WEB_TEMPLATE_PORT" and add  "VITE_WEB_PLANNER_PORT"**

File: apps/server-planner/logger.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  log(message: string) {
+    console.log(`\x1b[33m[${this.loggerName}]\x1b[0m ${message}`);
+  }
+
+  error(message: string) {
+    console.log("\x1b[31m%s\x1b[0m", `[${this.loggerName}] (ERROR) ${message}`);
+  }
+}
+
+export const logger = new Logger("SERVER-TEMPLATE");
```
Comment (vitormiura) [inline apps/server-planner/logger.ts:17]: needs to change to "SERVER-TEMPLATE" to PLANNER

**Comment by iguoliveira** [comment]
> **Forgot to change web and server ports, for server, need to change the PORT in index.ts to process.env.VITE_WEB_PLANNER_PORT | | 3002.** image **Is good too to setup cors too with planner web env, just add "VITE_WEB_PLANNER_PORT" instead of "VITE_WEB_TEMPLATE_PORT"** image **In web is necessary to change the port too, remove "VITE_WEB_TEMPLATE_PORT" and add "VITE_WEB_PLANNER_PORT" in vite.config.ts file** image

First of all, thanks for the clear instructions to how fix the bugs
image
image
Probably just a misspell, but in index.ts, needed to change from template to service-planner in PORT, for services run in 4002, however, working fine here

**Comment by abehidek** [comment]
CI passing now, needed to fix some imports, environment variables and config
I tested on my machine and it seems it's working fine on a docker dev environment.

Apparently, CI was failing before cause `web-planner` was importing `AppRouter` type from `server-template`, and that seems to trigger a build from `server-template`, however, `pnpm run build` cmd don't cover `server-template` because it does not have a `scripts` property in it's `package.json` file (it's named `_scripts` intentionally to not trigger build), I fixed the import to point to `server-planner` instead, then after fixing the unused variable by prefixing it with `_`, the build command seems to be working nicely.

I also did change the `.gitignore` file to ignore only the root dir `.env`file by prefixing it with `/`, this makes possible to commit the `.env` symlink files located inside the apps without forcing it.

Other than that, I also fixed some `cors` config to point to `web-planner` url instead of `web-template` url, and made the trpc client on `web-planner` point to `server-planner` url instead of `server-template` url

**Review by abehidek — APPROVED** [review]
Apparently it works fine now

**Comment by abehidek** [comment]
@vitormiura @viktormarinho please review it once again to see if there's any other fix to do.

**Review by vitormiura — APPROVED** [review]
Everything working fine, merging rn.

**Pull request merged**